ambry-api/src/main/java/com/github/ambry/config/NetworkConfig.java

```diff
@@ -119,6 +122,35 @@ public class NetworkConfig {
   @Default("false")
   public final boolean selectorUseDirectBuffers;
 
+  /**
+   * The type of queue to use to hold requests waiting to be processed.
+   * TODO: Do tests to tune controlled delay queue parameters before switching the request queue type to {@link RequestQueueType#ADAPTIVE_LIFO_CO_DEL}
+   */
+  @Config("request.queue.type")
+  @Default("BASIC_FIFO")
```

Comment:
Nit: just fifo would do unless you have an advance fifo and a basic fifo

Reply:
Also these defaults are not useful anywhere

Reply:
Actually meant to name as basic queue of type FIFO. Renamed it. Hopefully it is less confusing now.